frontend/src/components/Forms/MineRecordForm.js

```diff
@@ -25,7 +25,7 @@ import {
   getConditionalCommodityOptions,
   getDisturbanceOptionHash,
   getCommodityOptionHash,
-  getMineStatusOptions,
+  getMineStatusDropdownOptions,
```

Comment:
I added it as this function because I didn't remove the getMineStatusOptions selector. That selector still exists and will get the list of all of the acceptable status triplicates. I named it DropDownOptions because this is specifically getting information formatted only for use by the antd cascader.

Reply:
Okay. That makes sense. Maybe I'm missing something, but why does the diff show you removed `getMineStatusOptions` then?